unitTests/ui/MissingTermsTest.ts

```diff
@@ -45,6 +50,22 @@ export function MissingTermsTest() {
           expect(missingTermsClickableSpy).toHaveBeenCalled();
         });
 
+        it('true it should log an analytic', function() {
```

Comment:
Not sure that description is clear, maybe:
should log an analytics event when clicked